spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -131,6 +140,11 @@ public ArmeriaReactiveWebServerFactory(ConfigurableListableBeanFactory beanFacto
 
     @Override
     public WebServer getWebServer(HttpHandler httpHandler) {
+        final int port = ensureValidPort(getPort());
+        if (armeriaWebServer != null && needsToReuseWebServer(port)) {
```

Comment:
If `armeriaWebServer` is not null and the port is different, it seems like the previously configured settings to `armeriaWebServer` are gone. Isn't it right?

Reply:
The previously configured settings remains.
I'll add some tests to check this.